data/changelog.txt

```diff
@@ -1,5 +1,70 @@
 # 0.F (2021-04-15)
 
+3974 files changed, 5440251 insertions(+), 3904330 deletions(-)
+10,463 commits
+~486 contributors
+
+New game entities (core): 4510
+Items: 987
+    228 misc items, 199 books, 196 articles of clothing, 161 guns and gun related items,
+    151 comestibles, 99 tools, 53 ammunition types
+Mapgen: 778
```

Comment:
```suggestion
Mapgen (overmap level): 778
```

Reply:
These are the entities involved in determining what goes where, hence mapgen.
The map traits are the entities that define what the properties of things on the map are.

Better names are welcome, but they're definitely not both "mapgen".

Reply:
OK. I'm not the best at naming things, but... map objects? Map components? Map entities? Traits, at least to me, indicates something that would be over the entire map (or at least most of it), not a particular tile.

Reply:
~~It's hard for me to make any sort of judgment call without seeing the thousand-foot view.~~

Having looked at the broader context a little bit, what about "Overmap components" and "Map components"?

Reply:
```suggestion
Overmap Components: 778
```

Reply:
It's not overmap components either, it's a mixture of overmap and map level entities.  The distinguishing characteristic is these are what determines what map elements go where, and they are only referenced during mapgen.